data/default_os_images.json

```diff
@@ -208,5 +70,29 @@
         "cpu_architecture": "s390x",
         "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.18/4.18.1/rhcos-4.18.1-s390x-live.s390x.iso",
         "version": "418.94.202501221327-0"
+    },
+    {
+        "openshift_version": "4.19",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.19.0-ec.3/rhcos-4.19.0-ec.3-x86_64-live-iso.x86_64.iso",
```

Comment:
Even though 4.19 will be supported by 2.13 when it releases, I don't think we should put pre-release content in the release branches.